### PR TITLE
World Color Survey Database

### PR DESCRIPTION
**Pull request by kirillkoncha**
add world color survey database

### CODE REVIEW

**Comment by agricolamz** [comment]
@kirillkoncha, please, rename pull request, describe, changes

**Comment by kirillkoncha** [comment]
> @kirillkoncha, please, rename pull request, describe, changes

изменил, посмотрите изменения пожалуйста

File: tests/testthat/test-wcs-feature.R [hunk 1]
```diff
@@ -0,0 +1,5 @@
+library(testthat)
+context("Tests for eurasianphonology.feature function")
```
Comment (agricolamz) [inline tests/testthat/test-wcs-feature.R]: wrong context, just remove this line

File: R/wcs_bk.R [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#'
+#' Data from The database of WCS Data Archives (\url{https://www1.icsi.berkeley.edu/wcs/data.html}). This dataset is created for \code{\link{wcs.feature}} function.
+#'
+#' @format A data frame with 32211 rows and 13 variables:
+#' \describe{
+#'   \item{id}{Language id}
+#'   \item{speaker_id}{Speaker id}
+#'   \item{term_abbr}{Term abbriviation}
+#'   \item{language_original}{Language name}
+#'   \item{language}{Another Language name}
```
Comment (agricolamz) [inline R/wcs_bk.R]: can you add more meaningful comments?

File: R/wcs.feature.R [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+#'
+#' wcs.feature("WCS")
+#'
+#' @export
+#'
+#'
+wcs.feature <- function(feature){
+  message("Don't forget to cite a source:
+
+http://www.icsi.berkeley.edu/wcs/data.html")
```
Comment (agricolamz) [inline R/wcs.feature.R:18]: can you add a proper citations?

**Review by agricolamz — CHANGES_REQUESTED** [review]
work more on documentation

**Pull request closed without merging**